core/templates/pages/exploration-editor-page/services/exploration-save.service.ts

```diff
@@ -205,7 +205,6 @@ angular.module('oppia').factory('ExplorationSaveService', [
             RouterService.onRefreshVersionHistory.emit({
               forceRefresh: true
             });
-            AlertsService.addSuccessMessage('Changes saved.', 5000);
```

Comment:
Disabled Alert Service as in the e2e video we found that that changes were saved successfully but the toast message didn't came up and so now we will now be implementing a different way to check whether changes are saved or not. For more Info Please look at the debugging doc linked to the "Proof that changes are correct"

Reply:
You can leave the message, that's useful for the creator, right? The point of our discussion was to not have the e2e test use the appearance/disappearance of this message as the signal of success, but I wasn't talking about changing the user-facing flow.

Reply:
P.S. Please add me to the assignees list if you need me to look at something, I almost missed this comment. Thanks!

Reply:
Done. Reactivated the Alert Service for the toast message.